test/e2e/dns.go

```diff
@@ -128,7 +128,8 @@ func createProbeCommand(namesToResolve []string, fileNamePrefix string) (string,
 func assertFilesExist(fileNames []string, fileDir string, pod *api.Pod, client *client.Client) {
 	var failed []string
 
-	expectNoError(wait.Poll(time.Second*2, time.Second*60, func() (bool, error) {
+	start := time.Now()
+	expectNoError(wait.Poll(time.Second*2, time.Minute*5, func() (bool, error) {
```

Comment:
Looking back over the history of this test, the entire duration of the test when it succeeds is narrowly between 25 and 40 seconds.  And it does a lot more than just this check.  Can we increase the timeout from 1 minute to 2 minutes first to check whether there's any discernable difference in flakyness (which I doubt there will be).  5 minutes seems highly  excessive.